spec/integration/posts_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts features', type: :feature do
+  describe "User's post index page" do
+    before(:each) do
+      @first_user = User.create(name: 'LaLa', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                                bio: 'Teacher from Mexico.')
+      @first_post = Post.create(author: @first_user, title: 'Hello', text: 'This is my first post')
+      @second_post = Post.create(author: @first_user, title: 'Hello2', text: 'This is my second post')
+      @third_post = Post.create(author: @first_user, title: 'Hello3', text: 'This is my third post')
+      @fourth_post = Post.create(author: @first_user, title: 'Hello4', text: 'This is my fourth post')
+      @fifth_post = Post.create(author: @first_user, title: 'Hello5', text: 'This is my fifth post')
+      @first_comment = Comment.create(post: @first_post, author: @first_user, text: 'Hi Lala!')
+    end
+
+    it "Displays all user's data" do
+      visit user_posts_path(@first_user)
+
+      expect(page).to have_content(@first_user.name)
+      expect(page).to have_css("img[src='#{@first_user.photo}']")
+      expect(page).to have_content(@first_user.posts_counter)
+      expect(page).to have_content(@first_post.text)
+      expect(page).to have_content(@second_post.text)
+      expect(page).to have_content(@third_post.text)
+      expect(page).to have_content(@first_comment.text)
+      expect(page).to have_content(@first_post.comments_counter)
+      expect(page).to have_content(@first_post.likes_counter)
+      expect(page).to have_selector('.pagination')
+    end
```

Comment:
- [ ] For the `Posts#Index` page, you forgot to write a test that confirms the application renders **_a post's title_** of a user when a user visits the `Posts#Index` page. Kindly write this test. 👍 😎 

![Screenshot 2023-07-26 181100](https://github.com/UbdaNam/Blogiverse/assets/77106962/ee4859a0-85aa-4bf2-9835-d6e44d551129)

Reply:
Thanks for reviewing my PR 😀